src/panel/direction/DirectionPanel.jsx

```diff
@@ -27,15 +27,7 @@ export default class DirectionPanel extends React.Component {
   constructor(props) {
     super(props);
 
-    Telemetry.add(Telemetry.ITINERARY_OPEN,
-      props.poi && Telemetry.buildInteractionData({
-        id: props.poi.id,
-        source: props.poi.meta ? props.poi.meta.source : props.poi.name,
-        template: 'simple',
-        zone: 'detail',
-        element: 'itinerary',
-      })
-    );
+    Telemetry.add(Telemetry.ITINERARY_OPEN);
```

Comment:
shouldn't we move this in a DidMount lifecycle ?

Reply:
It will be called the same number of times here, as component instances are re-instantiated before each mount, but I agree it would be cleaner. I'll move it.